### PR TITLE
Adds http search audit route to Kibana

### PR DESCRIPTION
**Pull request by mejarrett**
https://rally1.rallydev.com/#/10660867787d/detail/userstory/156595869908

- Removes the restangular call to the old search audit route and changes it to the new http route (/api/audit/search). 
- Fixes indentation in the file.

Goes with www PR: https://github.schq.secious.com/Logrhythm/www/pull/1325

### CODE REVIEW

File: src/kibana/netmon_libs/custom_modules/save_rule/services/searchAuditor.js [hunk 1]
```diff
@@ -1,94 +1,90 @@
...
+   var angular = require('angular');
+   var timefilter = require('components/timefilter/timefilter');
 
-    var timefilter = require('components/timefilter/timefilter');
-    app.factory('searchAuditor', function (elasticSearchFields,
-      Restangular, timefilter) {
-         var FORMAT_STRING = 'YYYY/MM/DD HH:mm:ss';
-         var INDEX_PATTERN = '[network_]YYYY_MM_DD';
+   app.factory('searchAuditor', function (elasticSearchFields, Restangular, timefilter, $http) {
+      var FORMAT_STRING = 'YYYY/MM/DD HH:mm:ss';
```
Comment (alexweltman) [inline src/kibana/netmon_libs/custom_modules/save_rule/services/searchAuditor.js]: please make these consts (I know... not your fault!)

File: src/kibana/netmon_libs/custom_modules/save_rule/services/searchAuditor.js [hunk 2]
```diff
@@ -1,94 +1,90 @@
...
+         };
 
-        return {
-            logAndCapitalize : function(query) {
-                return elasticSearchFields.fetchMapping().then( function(hasFieldMap) {
-                    var capitalizedQuery = formatQuery(query.query_string.query)[0];
-                    logQuery(capitalizedQuery);
-                    // seems like we need to create a new object to force angular to reflect the
-                    // query change in the view
+         $http.put('/api/audit/search',
```
Comment (alexweltman) [inline src/kibana/netmon_libs/custom_modules/save_rule/services/searchAuditor.js]: the API route can be a const at the top with the others. 

weird formatting makes it hard to read... Maybe try something like this?
```JavaScript        
         $http.put(SEARCH_URI,  { 
            query: query,
            from: timeFrom,
            to: timeTo
         })
         .then(
             function (data) { }, 
             function (error) {
                var error = response.message || 'An unknown error occured';
                console.log('Search Audit Error:', error);
             }
         );
```

**Comment by craig-cogdill** [comment]
👍  After Alex's formatting comment. Code review only

File: src/kibana/netmon_libs/custom_modules/save_rule/services/searchAuditor.js (hunk 1 above)
Comment (mejarrett) [reply src/kibana/netmon_libs/custom_modules/save_rule/services/searchAuditor.js]: Np, done!

File: src/kibana/netmon_libs/custom_modules/save_rule/services/searchAuditor.js (hunk 2 above)
Comment (mejarrett) [reply src/kibana/netmon_libs/custom_modules/save_rule/services/searchAuditor.js]: Changed the route to a const, and beautified the formatting.

**Comment by alexweltman** [comment]
👍

**Pull request merged**